ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/exceptions/LdesShaclValidationException.java

```diff
@@ -1,16 +1,25 @@
 package be.vlaanderen.informatievlaanderen.ldes.server.domain.exceptions;
 
+import org.apache.jena.rdf.model.Model;
+
 public class LdesShaclValidationException extends RuntimeException {
 
 	private final String validationReport;
+	private final Model validationReportModel;
```

Comment:
when/where are you using the model?

Reply:
We have two flows. 
- The "/" endpoint that either returns the dcat or a http 500 without information.
- The "/admin/.../dcat" that either returns the dcat or a http 500 with a validation report. The validation report that we return here when status 500, is RDF.